.github/workflows/tag-release.yaml

```diff
@@ -0,0 +1,46 @@
+name: Tag a Build
+
+on:
+  # schedule event triggers always run on the default branch
+  # https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule
+  schedule:
+    # run "nightly" builds on default branch every mon/wed/fri
+    - cron: "21 2 * * 2,4,6"     # 2:21am UTC tues/thurs/sat == 7:21pm PDT mon/wed/fri -- see https://crontab.guru/#21_01_*_*_2,4,6
```

Comment:
github docs recommend not running scheduled dispatch near the top of the hour due to congestion, so I picked a minute with `python3 -c 'import random; print(random.randint(10,50))'`